src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1082,7 +1219,7 @@ public virtual AcceptedAtRouteResult AcceptedAtRoute(object routeValues)
         /// <summary>
         /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an Accepted (202) response.
         /// </summary>
-        /// <param name="routeName">The name of the route to use for generating the URL.</param>       
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
```

Comment:
I was thanking you here. Removing trailing whitespace in a file you're changing is fine.